pywal/__main__.py

```diff
@@ -15,11 +15,22 @@
 import shutil
 import sys
 
-try:
-    import colorama
-    colorama.just_fix_windows_console()
-except ImportError:
-    colorama = None
+if sys.platform.startswith("win"):
```

Comment:
I think the line above needs:

```python
show_colorama_warning = False
```

Reply:
done in https://github.com/eylles/pywal16/pull/152/commits/67af8f0662e1528f943dd4e2bd295a41b03c5302